### PR TITLE
Milestone 3: Actions handlers

### PR DESCRIPTION
**Pull request by tiagoit**
In this milestone, we connect the CREATE_BOOK and REMOVE_BOOK actions with the React component and make our page dynamic.

### CODE REVIEW

File: src/containers/BooksForm.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      title: event.target.value,
+    });
+  }
+
+  handleChangeSelect(event) {
+    this.setState({
+      category: event.target.value,
+    });
+  }
+
```
Comment (shloch) [inline src/containers/BooksForm.js]: You can condense these 2 in to ONE (see below) :
```
handleChange(e) {
    this.setState({
      [e.target.name]: e.target.value
    });
  };
```
and then bind the function to both `input` and `select` like this :
`onChange={this.handleChange.bind(this)}`

File: src/containers/BooksForm.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import PropTypes from 'prop-types';
+
+const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+class BookForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: '',
```
Comment (shloch) [inline src/containers/BooksForm.js]: i think there should be a default category to this state instead of an empty string

Comment (shloch) [reply src/containers/BooksForm.js]: Anyways, you can probably handle it another way 😊

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @tiagoit , @AnthonyTC89 
It's all good, works perfectly: 

## [changes required]
There's one improvement i made below to knock off 2 functions into one. Please implement that.
For the default category, this is optional, you can certainly do it another way. Hope you thought about that

Waiting for a last re-review hopefully

Happy coding 😊

File: src/containers/BooksForm.js (hunk 1 above)
Comment (AnthonyTC89) [reply src/containers/BooksForm.js]: Good idea! I like it!

File: src/containers/BooksForm.js [hunk 3]
```diff
@@ -9,22 +9,17 @@ class BookForm extends React.Component {
     super(props);
     this.state = {
       title: '',
-      category: '',
+      category: 'Action',
     };
     this.handleChangeInput = this.handleChangeInput.bind(this);
     this.handleChangeSelect = this.handleChangeSelect.bind(this);
```
Comment (shloch) [inline src/containers/BooksForm.js]: I guess you have to delete these 2 lines

**Review by shloch — APPROVED** [review]
## hello @tiagoit , @AnthonyTC89 
nice job
Please take note of comment below.
Then you can merge, happy coding

**Pull request merged**